ci/build-test-ios.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
don't think we need this script. we can just call the build ios client script directly in TeamCity.

Reply:
yes, that script is mainly for ci afaik. If you want to build your client, you would usually do it from the unity menu. 
so, just adding an additional line to the build-test script to build an ios client if on a mac agent should be enough